src/device.rs

```diff
@@ -213,12 +228,57 @@ pub extern "C" fn wgpu_texture_view_destroy(texture_view_id: id::TextureViewId)
     gfx_select!(texture_view_id => GLOBAL.texture_view_destroy(texture_view_id))
 }
 
+#[repr(C)]
+pub struct AnisotropicSamplerDescriptorExt {
+    pub next_in_chain: *const ChainedStruct,
```

Comment:
Can we instead have `Option<&ChainedStruct>` here and everywhere else?

Reply:
Should be all fixed. Worth a look inside the macro one more time to make sure I didn't accidentally any UB.